neon-processor/src/main/java/com/zing/zalo/neon2/internal/NeonConfig.java

```diff
@@ -0,0 +1,10 @@
+package com.zing.zalo.neon2.internal;
+
+/**
+ * Created by Tien Loc Bui on 17/09/2019.
+ */
+public final class NeonConfig {
+    public static final String INDENT_SPACE = "    ";
+    public static final String NEON_CLASS_PREFIX = "$Neon";
```

Comment:
nit: I'd prefer "$Neon__" and ""

Reply:
INDENT_SPACE "    " make code look better than ""